domain/src/main/java/org/oppia/android/domain/question/QuestionAssessmentProgressController.kt

```diff
@@ -615,7 +617,11 @@ class QuestionAssessmentProgressController @Inject constructor(
   }
 
   private fun ControllerState.computeBaseCurrentEphemeralState(): EphemeralState =
```

Comment:
Nit: please switch to block notation (we only use expression syntax for one and two liner methods).

Reply:
Still unresolved.

Reply:
done